service/storage_service.cc

```diff
@@ -3864,6 +3865,7 @@ future<> storage_service::drain_on_shutdown() {
 
 future<> storage_service::init_messaging_service_part(sharded<db::system_distributed_keyspace>& sys_dist_ks, bool raft_topology_change_enabled) {
     return container().invoke_on_all([&sys_dist_ks, raft_topology_change_enabled] (storage_service& local) {
+        create_session(service::session_id{}); // Default session.
```

Comment:
What default session is for?

Reply:
To be able to create a valid guard for streaming ops which don't use sessions created by topology code. If we have a default session, we can avoid special-casing for null guard.